### ISSUE TITLE
appcenter_upload to stores not working

### ISSUE THREAD

**Issue by jwilber99** [microsoft/fastlane-plugin-appcenter#311]
TL;DR - What are the values I can put in for the argument of 'destinations' when the 'destination_type' argument is 'store' instead of 'group'. I don't see example code for 'destinations' other than when it is being used for groups and the one linked below has the store name as 'Alpha', which gives me an error when I try.

Hello all,

I have been trying to use appcenter_upload() in my fastfile to upload our builds to the appcenter and the play / app store. It successfully uploads to the appcenter but it doesn't successfully upload to the stores. It tells me 'Not found, invalid distribution store name, store 'App Store' was not found'.

The arguments I am using look like this: appcenter_upload( file: options[:BUILD_OUTPUT], destinations: "App store", destination_type: "store", upload_build_only: true)

It doesn't recognize 'App store' as a name for a store. I have looked through the documentation and this one (link below) shows the 'destinations' as Alpha. I did try alpha but it tells me 'Not found, invalid distribution store name, store 'Alpha' was not found'. Example can be found above and at the link below (Link: [enter image description here](https://i.stack.imgur.com/M0rCP.png))

In the 'fastlane-lugin-appcenter's readme, it says that 'Comma separated list of destination names, use '*' for all distribution groups if destination type is 'group'. Both distribution groups and stores are supported. All names are required to be of the same destination type (default: Collaborators)'

What I am trying to accomplish: upload our builds to the app center (already done, have multiple builds in there) AND have it be uploaded to both the Google Play Console and the App Store Connect.

The main question I have is: What are the values that can be used for 'destinations' when we are trying to upload to a store and not a group of testers within AppCenter?

Fastlane action documentation (repo): https://github.com/microsoft/fastlane-plugin-appcenter/tree/f46cb07ae3fad3cadd57e349c662f80cd769121f

I tried using the arguments suggested here: https://github.com/microsoft/fastlane-plugin-appcenter/blob/f46cb07ae3fad3cadd57e349c662f80cd769121f/lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb#L709-L716

It successfully uploads to the app center, but it doesn't recognize the store name so it doesn't upload to the store. We already connected our team's store accounts from within the AppCenter. It is being uploaded through Jenkins so I am not directly uploading it so I don't think that I would need to login with my credentials to send the build.

I expected to have the build upload to the app center (already done) AND have the build upload to the stores as well. The documentation states that I need to simply state the 'destination_type: store' and the 'destinations' as the store's name.

What actually happened was that it uploads only to the app center and it doesn't recognize the store's name so it doesn't upload to the stores. We are trying to automate the process but we are blocked on this step specifically and there isn't too much documentation related to it. (specifically uploading to the stores AS WELL as uploading to the app center.) The readme and example documentation states that it is possible but I can't get it to work so far. Any help would be VERY appreciated!!

**Comment by MikhailSuendukov** [comment]
Hi @jwilber99 , sorry for the log response, did you manage to fix the error?
